installing/disconnected_install/installing-mirroring-disconnected.adoc

```diff
@@ -0,0 +1,125 @@
+:_content-type: ASSEMBLY
+[id="installing-mirroring-disconnected"]
+= Mirroring images for a disconnected installation using the oc-mirror plug-in
+include::modules/common-attributes.adoc[]
+:context: installing-mirroring-disconnected
+
+toc::[]
+
+////
+General to do list:
+* Confirm that no new content was added since I did the reorg of the disconnected install assemblies (up to date as of 2/13)
+* Add info about mirroring to a namespace
+* Test and get some actual example output
+* Use cases for changing image set config between runs
+////
+
+// TODO: This was pulled from the other existing assembly
+You can use the procedures in this section to ensure your clusters only use container images that satisfy your organizational controls on external content. Before you install a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. To mirror container images, you must have a registry for mirroring.
+
+You can use the oc-mirror OpenShift CLI (`oc`) plug-in to mirror images to a mirror registry in your fully or partially disconnected environments.
+
+:FeatureName: Mirroring images for disconnected environments using the oc-mirror plug-in
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+The following steps list the high-level workflow for using the oc-mirror plug-in to mirror images to a mirror registry:
+
+. Create an image set configuration file.
+. Mirror the image set to the mirror registry.
+. Installing the `ImageContentSourcePolicy` and `CatalogSource` resources into the cluster.
+. Repeat these steps to update your mirror registry as necessary.
+
+// About the oc-mirror plug-in
+include::modules/oc-mirror-about.adoc[leveloffset=+1]
+
+// TODO: Check the content of this module to make sure it's accurage for oc-mirror
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
```

Comment:
@afflom @jpower432 I pulled in this module from the existing docs, which talks about the mirror registry. Can you let me know if this makes sense to have in our section as well?

Preview here: https://deploy-preview-40474--osdocs.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#installation-about-mirror-registry_installing-mirroring-disconnected

Reply:
I think it does, but we have not verified compatibility with Artificatory, Nexus, or Harbor in the initial release of `oc-mirror`. Our main targets initially were Quay and Docker Registry.

Reply:
Okay so we do also list the registries in the prereqs section here: https://deploy-preview-40474--osdocs.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#prerequisites_installing-mirroring-disconnected

Should I update these places to only list Quay 3.6+ and Docker Registry? And remove  Artificatory, Nexus, and Harbor?

I'm assuming that the mirror registry for Red Hat OpenShift is still fine though, correct?

Reply:
1. mirror registry for openshift is compatible
2. I'd like to get feedback from users trying to integrate other registries with oc-mirror. Can we annotate untested integrations as not supported, but as something that we want feedback on?

Reply:
@afflom thoughts?

Reply:
Ignore my recent comments, I submitted before refreshing ;)